src/Pact/Repl/Lib.hs

```diff
@@ -451,12 +451,19 @@ verify i as = case as of
     uri <- fromMaybe "localhost" <$> viewLibState (view rlsVerifyUri)
     renderedLines <- liftIO $
                      RemoteClient.verifyModule modules md uri
+    setop $ TcErrors $ Text.unpack <$> renderedLines
```

Comment:
I see that we're not _always_ setting `TcErrors` for local verification now, but we are still always setting `TcErrors` for remote verification. Is this okay?

Reply:
Yeah, that's intentional. I think the commit message on https://github.com/kadena-io/pact/pull/551/commits/0f04b1f38cd337935149edc2809421f8525009ce explains

Reply:
Perfect -- thanks!